BOOM/Code/Services/audio_service.lua

```diff
@@ -0,0 +1,62 @@
+local ConfigService = require("Code.Services.config_service")
+
+local AudioService = {}
+local SFXAssets = {}
+local MusicAssets = {}
+
+local MUSIC_CHANNEL = 20
+
+------------------------------------------------------------------------------------
+-- Services
+------------------------------------------------------------------------------------
+function AudioService:oneShot(configId)
+    local config = ConfigService:fromId(configId)
+    local asset = config.asset
+    if type(asset) == "table" then
+        asset = asset[math.random(#asset)]
+    end
+
+    if asset then
+        local c = audio.findFreeChannel()
+        audio.play(
+            SFXAssets[asset],
+            {
+                channel = c,
+                loops = 0,
+                fadein = 0,
+            }
+        )
+    end
+end
+
+function AudioService:playMusic(configId)
+    local asset = MusicConfig[configId]
```

Comment:
`MusicConfig` is undefined

Reply:
Fixed this. Was using the old method of Config loading before I created the ConfigService